File: su/src/main/java/org/sqlbuilder/su/SuSynset31Resolver.java
package org.sqlbuilder.su;

import org.sqlbuilder.common.Resolver2;
import org.sqlbuilder2.ser.DeSerialize;

import java.io.File;
import java.io.IOException;

public class SuSynset31Resolver extends Resolver2<Character, Long, Long>
{
	public SuSynset31Resolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: su/src/main/java/org/sqlbuilder/su/SuSynset31Resolver.kt
package org.sqlbuilder.su

import org.sqlbuilder.common.DeSerialize
import org.sqlbuilder.common.Resolver2
import java.io.File

class SuSynset31Resolver(ser: String) : Resolver2<Char, Long, Long>(DeSerialize.deserialize<Map<Char, Map<Long, Long>>>(File(ser)))

